neuralmonkey/model/model_part.py

```diff
@@ -25,20 +26,34 @@ def __init__(self,
 
         self._saver = None  # type: Optional[tf.train.Saver]
 
+        with tf.variable_scope(name) as scope:
+            self._variable_scope = scope
```

Comment:
asi je to fuk.. Myslel jsem spíš, že by to šlo na jeden řádek jako
```python
self._variable_scope = tf.VariableScope(name)
```
ale jen vim, že takhle to nejspíš nefunguje.